offerzen/global/employers/prospects@2.2.js

```diff
@@ -0,0 +1,148 @@
+window.$loaded(function () {
+  window.$parsleyLoaded = function (cb) {
+    setTimeout(() => {
+      if (window.Parsley && window.ParsleyValidator) {
+        cb(window, document, parsley, undefined);
+        return;
+      }
+      $parsleyLoaded(cb);
+    }, 50);
+  };
+  window.$parsleyLoaded(function (window, document, parsley) {
+    function retrieveProspectJourney() {
+      if ('emailValue' in localStorage && 'acceptedTerms' in localStorage) {
+        // Redirect to book a call
+        window.location.href = '/employers/book-a-call';
+      }
+    }
+
+    function storeEmailAndTermsOfServiceValues() {
+      // Store email
+      localStorage.setItem('emailValue', emailValue);
+
+      // Store terms of service
+      if ($('#accepted_terms_of_service').is(':checked')) {
+        localStorage.setItem('acceptedTerms', 'on');
+      } else {
+        localStorage.setItem('acceptedTerms', 'off');
+      }
+    }
+
+    function onSubmitSingleStepProspectForm(token, e) {
+      const singleStepProspectForm = $('#Company-Lead-Form-Email-Only');
+      singleStepProspectForm.find('input[type=submit]').attr('disabled', true);
+      let initialButtonValue = singleStepProspectForm
+        .find('input[type=submit]')
+        .attr('value');
+      let dataWait = singleStepProspectForm
+        .find('input[type=submit]')
+        .attr('data-wait');
```

Comment:
Please use a variable for `singleStepProspectForm.find('input[type=submit]')` as above